src/ar.erl

```diff
@@ -227,6 +232,9 @@ start(
 	error_logger:logfile({open, Filename = generate_logfile_name()}),
 	error_logger:tty(false),
 
+	ar_meta_db:start(),
+	ar_meta_db:put(data_dir, DataDir),
```

Comment:
I would prefer if all `ar_meta_db:put/2` calls are moved up here so they stay together.

Btw, our style guide doesn't say anything about blank lines but I would like to not have any.

Reply:
I like the visual separation of the blocks, but comments can often be used for that. I am ok with not using blank lines.

Reply:
+1 for comments

Reply:
✅